### DISCUSSION TITLE
Difficulty in verifying my application

### DISCUSSION THREAD

**Discussion by mohammedyounes98** [community/community#116933] [GitHub Education]
### Hi everyone,

Hello,
I'm trying to validate my application for the student package, but I have several issues:

- I'm doing an internship in a different city away from my university, so I can't verify my location for the application and there's no alternative button or something to bypass this step as an error keeps telling me I'm in a different location.

- The only option is to take a picture, there's no way to upload documents? I mean the documents issued by my university online, and the document required to verify my internship is the agreement which is several pages long.
- When taking a picture using either Google Chrome or Safari on my iPhone, only the front camera works and the other million options of cameras are not working, making it harder to take a picture which won't even be clear.

Looking forward for your help!

Thanks

**Comment by User132New** [comment]
1. Use an app that could help you set a particular location as current location.
2. Take printout for your document page, which contains most essential details.
3. use Cast to see what's on screen and capture the photo.

**Comment by samus-aran** [comment]
Hi @mohammedyounes98 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
